public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -52,6 +55,16 @@ const evaluateEveryValidationOptions: RegisterOptions = {
   },
 };
 
+// TODO Move to more generic place
+export function useEvaluationIntervalGlobalLimit(alertGroupEvaluateEvery: string) {
```

Comment:
Maybe better `getEvaluationIntervalGlobalLimit` instead of `useEvaluationIntervalGlobalLimit` as this is not a hook?

Reply:
Yes, of course!